src/NServiceBus.Core.Tests/Config/When_no_custom_configuration_source_is_specified.cs

```diff
@@ -13,6 +13,7 @@ public class When_no_custom_configuration_source_is_specified
         {
             var config = new EndpointConfiguration("myendpoint");
 
+            config.SendFailedMessagesTo("error");
```

Comment:
Because we still run the those tests with MSMQ we need that to avoid blowing up. Once the learning transport is merged it can be removed